### DISCUSSION TITLE
Multiple Servers one Host

### DISCUSSION THREAD

**Discussion by Dimentagon** [apohl79/audiogridder#324] [Help]
Hi 
The short question is can I run and access multiple servers (with different id's) and access them from one host?

I have Audiogridder (the latest beta) running beautifully on a Mac Mini M1 with Logic, Studio One and Bitwig as a plug in host talking to an Intel i7 NUC running Windows 10 Pro as a server via an Ethernet hub. 

I’m getting 5ms in Bitwig 3 & Presonus Studio One 5 however, 20-30ms in Logic Audio? 

I tried to connect my 2015 MacBook Pro i7 (Big Sur thunderbolt to Ethernet) as a second server and cannot for the life of me get it to show up on the Mac Mini M1.

I’ve tried Add server with no luck. What am I doing wrong? 

I can share drives between the Mac mini M1 and the MacBook Pro but can’t get Audiogridder working as a second server option on the Mac Mini M1 despite it seeing all my plugs on the MacBook Pro. 

I’ve assigned a different server ID to the MacBook but no joy. The Windows 10 server is working like a charm but I'm lost as to why the MacBook is not appearing as an option on the M1.
Any help would be appreciated. 
thanks 
Ricky

**Comment by apohl79** [comment]
> can I run and access multiple servers (with different id's) and access them from one host?

When you say access them from one host, I assume you want to connect different AG plugins in your DAW to different AG server machines. That is possible.

> 20-30ms in Logic Audio

Maybe logic is configured to use a bigger I/O buffer. One complete I/O buffer defines a block in AG. The larger the buffer (in samples) the higher the latency added by a block.

> I’ve tried Add server with no luck. What am I doing wrong?

Can you ping the name or IP that you added? What is the error in the plugin logfile? (https://audiogridder.com/bug-report-diagnostics-locations/)

> despite it seeing all my plugs on the MacBook Pro

What do you mean by that? Do you see the remote plugins within the AG plugin when pressing "+" ? If so, your are connected.

> I’ve assigned a different server ID to the MacBook but no joy

You need different IDs only when running multiple AG servers on the same machine.

If you don't get it working, please follow the bug reporting steps (https://audiogridder.com/bug-reports/) and provide the plugin and server logs and I will take a look.
